src/main/java/com/example/dto/UserDto.java

```diff
@@ -0,0 +1,68 @@
+package com.example.dto;
+
+public class UserDto {
+
+    private int id;
+    private String firstName;
+    private String lastName;
+    private String username;
+    private String password;
+    private int age;
+    private int salary;
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public void setFirstName(String firstName) {
```

Comment:
final (if you don't believe me you can ask to another more qualifier programmer, that it's really necessary)